lutris/util/wine/wine.py

```diff
@@ -96,6 +96,22 @@ def is_gstreamer_build(wine_path: str) -> bool:
     return system.path_exists(os.path.join(base_path, "lib64/gstreamer-1.0"))
 
 
+def is_winewayland_available(runner_version: str) -> bool:
```

Comment:
Hmm. No sign of my fancy Proton version number check.

Will this check for Wayland .drv files work? It should be True for GE-Proton10-1 and later but false for GE-Proton9.

Reply:
Yeah, I think it should be fine, because regular builds of Proton don't have winewayland files, and older Proton-GE versions don't either AFAIK.

Reply:
I checked Proton-Tkg-2074 and it does have those files.

Reply:
> I checked Proton-Tkg-2074 and it does have those files.

Yes, Proton-TKG also has winewayland. I think they were one of the first to attempt to use it, although I'm not sure if they use `PROTON_ENABLE_WAYLAND` variable.

Reply:
Hmm, looks like Proton-TKG doesn't have `PROTON_ENABLE_WAYLAND` variable :\

Maybe we should also add a `DISPLAY=` variable as a safety precaution, but this needs to be tested.

Reply:
If they do not, how do you activate the feature? With the registry key?

We need to figure out which method to use for which Wine build, somehow.

Reply:
> If they do not, how do you activate the feature? With the registry key?
> 
> We need to figure out which method to use for which Wine build, somehow.

Nope, from what I read in this issue https://github.com/Frogging-Family/wine-tkg-git/issues/1425, the only way to activate it is to unset `DISPLAY` environment variable, as it works for the current upstream as well. Not sure if this will work for Proton-TKG versions based on Proton 9.

Reply:
This is getting complicated! We probably can't handle every custom Proton build; I'd rather handle what we ship (and that's GE-Proton and wine-ge).

Reply:
And ``wine-ge`` does not support Wayland at all (it is too old) so hey presto now my PR is the way to go!

Maybe we should support vanilla Wine builds with Wayland too, though. We don't ship them, but we'll use them if installed.

Reply:
Seems to be Proton-TKG just added support for the `PROTON_ENABLE_WAYLAND` variable, so I think this is solved now, probably for all known Proton builds. 

https://github.com/Frogging-Family/wine-tkg-git/commit/f4ee327a19cb06c9a7c7502c84a7e62b65e0fc14